Makefile

```diff
@@ -70,6 +73,10 @@ stablerepo: $(STABLE_TARGETS) | gh-pages/stable/index.yaml
 .PHONY: publish
 publish: export LC_COLLATE := C
 publish:
+ifeq ($(PLATFORM),darwin)
+	$(warning The publish task uses GNU executables and macOS ships with BSD ones by default.)
```

Comment:
Can we name the specific tools that I'd need to check as a user to make sure they are the GNU ones?

Reply:
Yeah it would be good to know and if we can't just update to use the BSD ones on macos, or validate and error if required GNU ones are unavailable?